components/Banner/Banner.css

```diff
@@ -0,0 +1,21 @@
+.banner {
+  border: 1px solid #DCDCDC;
+  background: #f3f3f3;
+  border-radius: 3px;
+  border-color: #dbdbdb;
+  padding: 8px 8px;
+  min-width: 200px;
+  font-size: 16px;
+}
```

Comment:
could use some space underneath of it

Reply:
I just added some padding in OrganizationEdit.css.  Here's the new look ![](https://content.screencast.com/users/pididiot1/folders/Jing/media/f3ac3533-af36-4d2a-ad75-188e39716668/00000143.png)